frontend/public/components/utils/nav-title.tsx

```diff
@@ -36,9 +36,12 @@ export const NavTitle = connectToModel((props: NavTitleProps) => {
   const {kind, kindObj, detail, title, menuActions, buttonActions, obj, breadcrumbsFor, style} = props;
   const data = _.get(obj, 'data');
   const isCSV = !_.isEmpty(data) && referenceFor(data) === referenceForModel(ClusterServiceVersionModel);
```

Comment:
I must have let this slip through on a previous review, but there should be no catalog specific logic here.

Reply:
Ok, seems like that should be handled in a follow on PR